app/models/work.rb

```diff
@@ -411,7 +423,10 @@ def save_new_recipients
     unless self.new_recipients.blank?
       self.new_recipients.split(',').each do |name|
         gift = self.gifts.for_name_or_byline(name).first
-        self.gifts << Gift.new(:recipient => name) unless gift
+        unless gift
```

Comment:
This might be a little easier to read if it was "if gift.nil?" or "unless gift.present?"